x-pack/plugins/canvas/common/lib/handlebars.js

```diff
@@ -5,12 +5,17 @@
  */
 
 import Hbars from 'handlebars/dist/handlebars';
+import { i18n } from '@kbn/i18n';
 import { evaluate } from 'tinymath';
 import { pivotObjectArray } from './pivot_object_array';
 
 // example use: {{math rows 'mean(price - cost)' 2}}
 Hbars.registerHelper('math', (rows, expression, precision) => {
-  if (!Array.isArray(rows)) return 'MATH ERROR: first argument must be an array';
+  if (!Array.isArray(rows)) {
+    return i18n.translate('xpack.canvas.handlebars.firstArgumentTypeErrorMessage', {
+      defaultMessage: 'MATH ERROR: first argument must be an array',
```

Comment:
Consider adding what type was found instead.
```suggestion
      defaultMessage: 'MATH ERROR: first argument must be an array; found {rowsType}',
      values: {rowsType: typeof rows},
```